checker/check-response-property-default-value-changed.go

```diff
@@ -0,0 +1,69 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func ResponsePropertyDefaultValueChangedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
+	result := make(Changes, 0)
+	if diffReport.PathsDiff == nil {
+		return result
+	}
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.OperationsDiff == nil {
+			continue
+		}
+		for operation, operationItem := range pathItem.OperationsDiff.Modified {
+			source := (*operationsSources)[operationItem.Revision]
+			if operationItem.ResponsesDiff == nil || operationItem.ResponsesDiff.Modified == nil {
+				continue
+			}
+
+			for responseStatus, responseDiff := range operationItem.ResponsesDiff.Modified {
+				if responseDiff.ContentDiff == nil ||
+					responseDiff.ContentDiff.MediaTypeModified == nil {
+					continue
+				}
+
+				modifiedMediaTypes := responseDiff.ContentDiff.MediaTypeModified
+				for mediaType, mediaTypeDiff := range modifiedMediaTypes {
+					if mediaTypeDiff.SchemaDiff != nil && mediaTypeDiff.SchemaDiff.DefaultDiff != nil {
+						defaultValueDiff := mediaTypeDiff.SchemaDiff.DefaultDiff
+						result = append(result, ApiChange{
+							Id:          "response-body-default-value-changed",
+							Level:       INFO,
+							Text:        fmt.Sprintf(config.i18n("response-body-default-value-changed"), ColorizedValue(mediaType), empty2none(defaultValueDiff.To), empty2none(defaultValueDiff.To), ColorizedValue(responseStatus)),
```

Comment:
first empty2none(defaultValueDiff.To) should be empty2none(defaultValueDiff.From)

Reply:
nice catch thanks, updating